lib/bolt/inventory.rb

```diff
@@ -73,31 +74,53 @@ def get_targets(targets)
       targets.map { |t| update_target(t) }
     end
 
-    # Should this be a public method?
     def config_for(node_name)
       data = @groups.data_for(node_name)
       if data
         Bolt::Util.symbolize_keys(data['config'])
       end
     end
 
+    def set_var(target, key, value)
+      data = { 'vars' => { key => value } }
```

Comment:
This needs to change to match the updated signature of `set_vars_from_inv`. I'd be in favor of changing the function name to `set_vars_from_hash` as well.

Reply:
I think this is the cause of the test failures.